scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerImages.java

```diff
@@ -39,7 +39,7 @@
 @SuppressWarnings("squid:S1214")
 public interface SchedulerImages extends ClientBundle {
 
-    public static final SchedulerImages instance = GWT.create(SchedulerImages.class);
+    SchedulerImages instance = GWT.create(SchedulerImages.class);
 
     @Source("images/cancel_14.png")
     ImageResource cancel_14();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)